code/nx.json

```diff
@@ -52,7 +47,10 @@
           "dependencies": true
         }
       ],
-      "outputs": ["{projectRoot}/dist"]
+      "outputs": ["{projectRoot}/dist"],
+      "cache": true
     }
-  }
+  },
+  "nxCloudAccessToken": "NGVmYTkxMmItYzY3OS00MjkxLTk1ZDktZDFmYTFmNmVlNGY4fHJlYWQ=",
```

Comment:
Is this OK to leave here?

Reply:
I don't know who has set up this initially. The value was just moved from the top config to the bottom here. NX says in their documentation that you should use a read-only token here and you could use a write token in your environment variables. https://nx.dev/nx-cloud/account/access-tokens#setting-a-different-access-token-in-ci. But if this one is read-only, everyone would have to set up a nx-cloud.env locally `https://nx.dev/nx-cloud/account/access-tokens#using` to be able to write to the cache